src/main/java/org/triplea/ui/TripleA.java

```diff
@@ -0,0 +1,104 @@
+package org.triplea.ui;
+
+import java.net.URL;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.Tooltip;
+import javafx.scene.image.Image;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+public class TripleA extends Application {
+
+  @FXML
+  private VBox loadingOverlay;
+
+  @FXML
+  private VBox exitOverlay;
+
+  @FXML
+  private Tooltip progressTooltip;
+
+  @FXML
+  private Label progressLabel;
+
+  @FXML
+  private StackPane rootPane;
+
+  private TripleaMainMenu mainMenu;
+
+  @Override
+  public void start(final Stage stage) throws Exception {
+    final FXMLLoader loader = getLoader(getClass().getResource("./fxml/TripleAMain.fxml"));
+    loader.setController(this);
+    final Scene scene = new Scene(loader.load(), 960, 540);
```

Comment:
Yes, of course.
But since we don't have such a config yet, I wanted to set some values so the window can't be resized smaller than this